setup-slave.sh

```diff
@@ -45,6 +45,21 @@ if [[ $instance_type == r3* ]]; then
   fi
 fi
 
+# Work around for hi1.4xlarge instances without pre-formatted ext3 disks
+if [[ $instance_type == hi1.4xlarge ]]; then
```

Comment:
Hmm I am not sure. Sometimes `/dev/sdc` is symlinked to `/dev/xvdf`. Do you have a cluster where you can try it with `sdc` and see if it works ?